algorithms/Python/linked_lists/singly.py

```diff
@@ -78,6 +78,51 @@ def pop(self):
             raise IndexError("pop from empty LinkedList")
         self.head = node.next
         return node.data
+    
+    def insertEnd(self, data):
+        '''
+        Insert node at the end of linked list
+        '''
+        node = Node(data)
+        if self.head is None:
+            self.head = node
+            return
+
+        # Linked list traversal
+        temp = self.head
+        while(temp.next is not None):
+            temp = temp.next
+
+        temp.next = node
+        return
+    
+    def deleteData(self, data):
+        '''
+        Delete first occurence of given data
```

Comment:
typo caught by code-spell : occurence ==> occurrence

```suggestion
        Delete first occurrence of given data
```